src/components/ha-icon-button-arrow-next.ts

```diff
@@ -36,7 +36,7 @@ export class HaIconButtonArrowNext extends LitElement {
   protected render(): TemplateResult {
     return html`<mwc-icon-button
       .disabled=${this.disabled}
-      .label=${this.label || this.hass.localize("ui.common.next")}
+      .label=${this.label || this.hass?.localize("ui.common.next")}
```

Comment:
Should we fallback to `|| "next"` here if no label and no hass?
(same goes for the other files)